content/operate/kubernetes/release-notes/7-4-6-releases/7-4-6-tbd-dec24.md

```diff
@@ -0,0 +1,32 @@
+---
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: This is a maintenance release to support Redis Enterprise Software version 7.4.6-102 and contains a suggested fix needed to upgrade to newer versions.
+hideListLinks: true
+linkTitle: 7.4.6-tbd (Dec 2024)
+title: Redis Enterprise for Kubernetes 7.4.6-tbd (Dec 2024) release notes
+weight: 1
+---
+
+## Highlights
+
+This is a maintenance release to support [Redis Enterprise Software version 7.4.6-102]({{<relref "operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-6-102">}}). This version also contains a suggested fix needed to upgrade to newer versions. For version changes, supported distributions, and known limitations, see the [release notes for 7-4-6-2 (July 2024)]({{<relref "/operate/kubernetes/release-notes/7-4-6-releases/7-4-6-2">}}).
```

Comment:
> This version also contains a suggested fix needed to upgrade to newer versions.

@yuvallevy2 - Does this fix need to be installed to upgrade to 7.8.2 or later? Or something else?

Reply:
@yuvallevy2 who can get us any changes needed around upgrade path?